src/sentry/eventtypes/error.py

```diff
@@ -11,7 +11,7 @@ def get_crash_location(data):
     from sentry.stacktraces.processing import get_crash_frame_from_event_data
 
     frame = get_crash_frame_from_event_data(
-        data, frame_filter=lambda x: x.get("filename") or x.get("abs_path")
+        data, frame_filter=lambda x: x.get("function") not in (None, "<redacted>", "<unknown>")
```

Comment:
This now means that the displayed crash title is no longer the one we match on for fingerprinting (`resolve_fingerprint_values`). I think this is not a good idea.

Reply:
Correction, they were already not matching. But I think this should probably be the same logic.

Reply:
`resolve_fingerprint_values` never passed a filter in the first place. How could that have ever matched?

Reply:
It will pick the first `in_app` frame.

Reply:
Ah sorry, my GH page was outdated; didn't see the second comment.

Yes, we can make it match, although that might change grouping now. I'd rather not change fingerprinting in this bugfix and reevaluate in a follow-up PR.